src/app/members/members-data.ts

```diff
@@ -798,4 +798,23 @@ export const members: TMember[] = [
       { name: "Weakness", value: 100, max: 100 },
     ],
   },
+  {
+    name: "Siddartha Bhandari",
+    login: "siddartha",
+    role: ["B3", "Current"],
+    imagePath: "images/members/rika.jpg",
```

Comment:
@SIDDHARTHABHANDARI It looks like you are pointing to Rika's picture here, and you have not added a picture of your own in this PR. Did you follow the instructions in the README and test this locally?